packages/manager-components/src/components/templates/update-name-modal/update-name-modal.component.tsx

```diff
@@ -91,10 +107,27 @@ export const UpdateNameModal: React.FC<UpdateNameModalProps> = ({
           disabled={isLoading || undefined}
           type={ODS_INPUT_TYPE.text}
           value={displayName}
+          error={pattern && isPatternError}
```

Comment:
```suggestion
          error={pattern && isPatternError || undefined}
```

If the ODS component still have the problem with the boolean attributes

Reply:
👍